R/PythonPkgWrapperUtils.R

```diff
@@ -197,14 +197,34 @@ autoGenerateFunctions <- function(setGenericCallback,
   }
 }
 
+# Helper function to camel case the given input
+#
+# @param x the input string
+snakeToCamel <- function(x) {
+  title <- function(x) {
+    paste0(
+      toupper(substring(x, 1, 1)),
```

Comment:
Why not replace this with title(x), as you introduced below?

Reply:
I think I might have been a little confused about your question during the review, but the title function was nested in the snakeToCamel function. I've pulled it out into a separate function which I think would make it clearer.

Reply:
My comment was that you had two different functions doing the same thing: The original one (I wrote) is the one you refactored to capitalizeFirstLetter().  The other was title().  I thought you would remove my one-off function in favor of using title() but the change you made is fine (because it eliminates the duplication of functionality).